modules/phase_field/src/userobjects/SolutionRasterizer.C

```diff
@@ -10,6 +10,8 @@
 #include "SolutionRasterizer.h"
 
 #include <fstream>
+#include "libmesh/mesh_function.h"
+#include "libmesh/exodusII_io.h"
```

Comment:
```suggestion
```
are you including those for the namespace? I dont see any use of these objects lower

Reply:
Without those includes I get incomplete type warnings from a unique pointer.